### ISSUE TITLE
Analysis: Compare Fixture data to Optimo payload via Ares

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#792]
**Is your feature request related to a problem? Please describe.**
Compare the local fixture data we have to a payload from Ares and discuss the difference and actions that need to be performed to align the two. 

**Describe the solution you'd like**
Compare the data and give clear breakdown of the differences. 

**Describe alternatives you've considered**
N/A

- [x] Intially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

**Comment by pjlee11** [comment]
Using the Ares test asset [c9rpqy7pmypo.json](https://www.test.bbc.co.uk/news/articles/c9rpqy7pmypo.json) compared to the fragmented Fixture data asset [c0000000025o.json](https://github.com/BBC-News/simorgh/blame/6b81b0c1bfd681d9fd7e01a448e3c05bcab805a3/data/fragmented/prod/news/c0000000025o.json) I did some analysis as the Ares test asset was created on Optimo as a replica of c0000000025o. 

Each comparison is alphabetically sorted and linted by http://novicelab.org/jsonabc/ and https://jsonlint.com/ respectively. Also in the diffchecker links the left side is the ares article and right side if the fixture asset.

[First comparison](https://www.diffchecker.com/aFpl5k3X)
  Action: Remove blockId's from fixture

[Second comparison](https://www.diffchecker.com/85mpyC4o)
  Action: Make links absolute in fixture

[Third comparison](https://www.diffchecker.com/Y26fBerM)
  Action: Remove image base endpoints

[Fourth comparison](https://www.diffchecker.com/EnBGo3w0)
  Action: Remove tags {}

[Fifth comparison](https://www.diffchecker.com/jf8X9rRQ)
  Action: Remove promoHeadline and seoHeadline

[Sixth comparison:](https://www.diffchecker.com/6qLgl2yn)
  Action: Remove empty caption block from ares test payload

[Seventh comparison](https://www.diffchecker.com/oLODjZpl)
  Action: Change ares payload `alt-text` to `altText`

[Final comparison](https://www.diffchecker.com/3VLSTaBY)

Above actions plus:
  - remove category
  - remove genre
  - remove created
  - add language
  - add options
  - add relatedContent block
  - add analyicsLabels
  - add 'News' to createdBy

Notes: 
  - alt-text can be ignored was a known issue
  - convince optimo not to output a completely empty caption block
  - originCode known to be missing

## TL;DR
We **need** to do the following to the fixture data:
- remove the blockIDs 
- make the urlLink's absolute
- remove the image basePath
- remove `category`, `genre` and `created`
- add `language`, `options`, `relatedContent` block, `analyticsLabels`
- update the schema to reflect the above

We also **need** to ask Optimo about removing empty caption blocks.

**Comment by pjlee11** [comment]
Image basePath https://github.com/BBC-News/simorgh/issues/793 and https://github.com/BBC-News/simorgh/issues/786

**Comment by pjlee11** [comment]
Remove blockId's https://github.com/BBC-News/simorgh/issues/794

**Comment by pjlee11** [comment]
urlLink's absolute - https://github.com/BBC-News/simorgh/issues/785 followed by https://github.com/BBC-News/simorgh/issues/784

**Comment by pjlee11** [comment]
remove/add values https://github.com/BBC-News/simorgh/issues/795
